src/Okta.Sdk/Generated/UserType.Generated.cs

```diff
@@ -40,11 +40,7 @@ public string DisplayName
         }
         
         /// <inheritdoc/>
-        public string Id 
-        {
-            get => GetStringProperty("id");
-            set => this["id"] = value;
-        }
+        public string Id => GetStringProperty("id");
```

Comment:
See previous comment; this may cause existing code to not compile.  Did you rebase?